internal/compiler-interface/src/main/contraband/other.json

```diff
@@ -28,6 +28,24 @@
             "Combined with a way to tell if the provenance has changed,",
             "it can be used to short-circuit the 'lookupAnalyzedClass' operation."
           ]
+        },
+        {
+          "name": "bytecodeHash",
+          "type": "int",
```

Comment:
We might as well allocate `long` even if we use only the `int` portions initially.